### PR TITLE
Add initial Japanese language files

### PR DESCRIPTION
**Pull request by kotaroman**
## Contribution Guidelines

* [x] I've read the [contribution guidelines](https://github.com/mailcow/mailcow-dockerized/blob/master/CONTRIBUTING.md) and wholeheartedly agree them

### Short Description

This PR adds initial Japanese language support for mailcow by introducing translation files in `lang/ja`. It aims to provide better accessibility for Japanese-speaking users.

###  Affected Containers

- None (This PR affects only language files, not specific containers)

### What did you tested?

- Verified that the Japanese language files load correctly in the application.
- Tested switching to Japanese and ensuring that translations are properly displayed.

### What were the final results? (Awaited, got)

- Awaited: Japanese translations should display correctly in the interface.
- Got: Translations worked as expected, and no issues were found.

スクリーンショット 2024-12-01 223448

### CODE REVIEW

File: data/web/inc/vars.inc.php [hunk 1]
```diff
@@ -110,6 +110,7 @@
   'uk-ua' => 'Українська (Ukrainian)',
   'zh-cn' => '简体中文 (Simplified Chinese)',
   'zh-tw' => '繁體中文 (Traditional Chinese)',
+  'ja-jp' => '日本語 (Japanese)',
```
Comment (MAGICCC) [inline data/web/inc/vars.inc.php]: Maybe we should move that up to Italian

**Review by MAGICCC — CHANGES_REQUESTED** [review]

File: data/web/inc/vars.inc.php (hunk 1 above)
Comment (kotaroman) [reply data/web/inc/vars.inc.php]: @MAGICCC 
Thank you for the suggestion! I have already updated the code to move Japanese (日本語) below Italian (Italiano). Please let me know if there are any further adjustments needed.

**Comment by kotaroman** [comment]
I have updated the PR to move Japanese (日本語) below Italian (Italiano) as suggested. Let me know if there are any other changes required.

スクリーンショット 2024-12-05 061442

**Review by MAGICCC — APPROVED** [review]

**Comment by MAGICCC** [comment]
@DerLinkman 
Looks ok, need to check how to integrate it into weblate once merged, prob only need to add the language inside weblate

**Comment by DerLinkman** [comment]
Thank you! Greetings to Japan :)

**Comment by kotaroman** [comment]
Thank you for merging the PR! I'm glad to contribute. I hope Mailcow will continue to grow! 😊
@DerLinkman @MAGICCC

**Pull request merged**